MdeModulePkg/Universal/Console/GraphicsConsoleDxe/GraphicsConsole.c

```diff
@@ -432,6 +432,8 @@ GraphicsConsoleControllerDriverStart (
 
   HorizontalResolution  = PcdGet32 (PcdVideoHorizontalResolution);
   VerticalResolution    = PcdGet32 (PcdVideoVerticalResolution);
+  HorizontalResolution  = 1024;
+  VerticalResolution    = 768;

```

Comment:
Can we define PcdVideoHorizontalResolution and PcdVideoVerticalResolution in OvmfPkgX64.dsc instead?

Reply:
Sure. It will be updated.